src/System.Private.Xml/src/System/Xml/Core/XmlConfiguration.cs

```diff
@@ -25,24 +25,8 @@ internal static class XmlConfigurationString
     }
 
     [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
-    internal sealed class XmlReaderSection : ConfigurationSection
+    internal sealed class XmlReaderSection
```

Comment:
Especially if we don't have any way to configure these things (not sure if that is true or not).